<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registerwindow</class>
 <widget class="QDialog" name="Registerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="password2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="usernameerror">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;*Already existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passworderror">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;*Not matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="dob">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sep</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oct</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dec</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="day">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="year">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="doberror">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;*Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="male">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="accounttype">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="user">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="genre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Favourite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="registrationerror">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;*All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
